proofs/lfsc/signatures/quantifiers_rules.plf

```diff
@@ -45,7 +45,5 @@
                       (! r (^ (sc_substitute body v s) bodyi)
                         (holds (exists v u body))))))))))
 
-(declare skolem_intro (! u term
-                      (! t term
-                      (! r (^ (sc_mk_skolem t) u)
-                        (holds (= u t))))))
+(declare skolem_intro (! t term
+                        (holds (= (skolem t) t))))
```

Comment:
what is this rule even checking then?

Reply:
Nothing, it doesnt need to check anything.

Reply:
Can't you then simplify the calculus to remove it?

Reply:
This will depend on the role of substitutions in LFSC.

`skolem` still acts as a way of telling substitutions that the given term is effectively a black box.

However, I am considering making it so that all substitutions are fully expanded before printing LFSC proofs.  Notice that this means special treatment would be needed for proofs of INSTANTIATE.

Of course, if this is done, this will be in a separate PR :)

Reply:
I see. :)